tc/src/main/java/com/epam/tc/service/DefaultValuePopulator.java

```diff
@@ -1,8 +1,10 @@
 package com.epam.tc.service;
 
+import com.epam.tc.model.Category;
 import com.epam.tc.model.Course;
 import com.epam.tc.model.User;
 import com.epam.tc.model.UserRole;
+import com.epam.tc.service.category.CategoryService;
 import com.epam.tc.service.course.CourseService;
 import com.epam.tc.service.evaluate.EvaluateService;
 import com.epam.tc.service.user.UserService;
```

Comment:
Such fake data creation, intialization classes should be in separate package - so do not mess with production code.

Also could you please rename class to something like SampleDataPopulator.

Some package name examples:
- com.epam.tc.init
- com.epam.tc.fake